source/src_lcao/ELEC_scf.cpp

```diff
@@ -552,9 +552,17 @@ void ELEC_scf::scf(const int &istep)
 					GlobalV::ofs_running << " " << GlobalV::global_out_dir << " final etot is " << GlobalC::en.etot * ModuleBase::Ry_to_eV << " eV" << std::endl;
 				}
 #ifdef __DEEPKS
-				if (INPUT.deepks_scf)	//caoyu add 2021-06-04
+				if (INPUT.out_descriptor)	//caoyu add 2021-06-04
```

Comment:
Please avoid to use "INPUT.", you can add a new variable in global_variable.h and use it for replace that.